src/collective/cover/config.py

```diff
@@ -25,7 +27,9 @@
     'News Item'
 ]
 
-DEFAULT_GRID_SYSTEM = 'deco16_grid'
+PLONE_VERSION = pkg_resources.require('Plone')[0].version
```

Comment:
This actually tries installing Plone, getting packages from the Internet if for some unimaginable reason Plone is not available yet. Sure, in practice that cannot happen, but it *does* try to do too much.  We should use this instead, which is a bit faster anyway:

    pkg_resources.get_distribution('Plone').version

See also the [plone.api docs](http://docs.plone.org/external/plone.api/docs/contribute/conventions.html#about-imports)